fragments/form/model.json

```diff
@@ -0,0 +1,51 @@
+{
+  "definitions": {
+    "Form": {
+      "type": "object",
+      "x-type": "component",
+      "properties": {
+        "schema": {
+          "type": "string",
+          "x-source": "inject",
+          "x-form-label": "Form Model",
+          "x-form-type": "textarea",
+          "x-form-hint": "JSON description passed to vue-form-generator, see <a href='https://vue-generators.gitbook.io/vue-generators/fields'>vue form generator fields documentation</a>",
```

Comment:
need to add a target to this so it opens up in a new tab

Reply:
Added _blank